crates/indexer/migrations/3_proof_requests.sql

```diff
@@ -4,7 +4,6 @@ CREATE TABLE IF NOT EXISTS proof_requests (
     client_address      TEXT      NOT NULL,  -- Ethereum address
     
     -- Requirements fields
-    image_id            TEXT      NOT NULL, -- 32-byte image ID (hex encoded)
```

Comment:
I'm in favor of this approach of keeping things clean. However this will require nuking the existing db as the migration will not be compatible. We could leave the value and use a dummy value as an alternative if we prefer keeping compatibility

Reply:
What is the consequence of nuking the indexer DB? I don't have a strong opinion of whether or not to keep this field to maintain compatibility. I do feel like we want to start indexing claim digests and callback data though.